### PR TITLE
Port: Metamorphic Glasses

### PR DESCRIPTION
**Pull request by OrbisAnima**
Basically, this just adds the old style glasses to the code without
removing anything.

https://i.imgur.com/eeqjW7c.png

Glasses can be found in the bar vendor, or can be ordered with the
bar/party supplies like any other glasses.

### CODE REVIEW

**Comment by OrbisAnima** [comment]
@Anewbe , its done, I did added them to the cargo crates too, but did not add them to the autolathes.

File: code/modules/reagents/reagent_containers/drinkingglass/metaglass.dm [hunk 1]
```diff
@@ -0,0 +1,502 @@
...
+	desc = "This glass changes shape and form depending on the drink inside... fancy!"
+	icon_state = "glass_empty"
+	amount_per_transfer_from_this = 5
+	volume = 30
+	unacidable = 1 //glass
+	center_of_mass = list("x"=16, "y"=10)
+	matter = list("glass" = 500)
+	icon = 'icons/obj/drinks.dmi'
+
+	on_reagent_change()
```
Comment (Anewbe) [inline code/modules/reagents/reagent_containers/drinkingglass/metaglass.dm]: Absolute pathing, please.

Comment (OrbisAnima) [reply code/modules/reagents/reagent_containers/drinkingglass/metaglass.dm]: roger

**Comment by OrbisAnima** [comment]
there we go

**Pull request merged**